x/hub-genesis/types/keys.go

```diff
@@ -13,7 +12,7 @@ const (
 	// QuerierRoute is the querier route for the minting store.
 	QuerierRoute = StoreKey
 
-	// Query endpoints supported by the minting querier.
+	// QueryParameters endpoints supported by the minting querier.
```

Comment:
`QueryParameters` is not used

Reply:
also `RouterKey` is not used

Reply:
ditto

Yes you're right it should be removed
but is it ok if I do it in a followup PR? This PR is currently all tested etc
https://github.com/dymensionxyz/dymension-rdk/issues/452
I have made an issue for myself
Thanks